ContextDesktop/src/main/java/com/github/migi_1/Context/main/LobbyHUDController.java

```diff
@@ -111,22 +111,30 @@ private void addInstruction() {
     public void setPlayerText(PlatformPosition position, String text) {
     	players.get(position).setText(text);
     }
-    
+
     /**
      * Sets the position of a text field.
      * @param text
      *              The text field.
      * @param widthFactor
      *              The width value at which the center of the text field
      *              will be placed.
-     * @param heightFactor 
+     * @param heightFactor
      *              The height value at which the text field will be placed.
      */
-    private void setTextPosition(BitmapText text, float widthFactor, 
+    private void setTextPosition(BitmapText text, float widthFactor,
             float heightFactor) {
         float width = widthFactor * settings.getWidth() - 0.5f * text.getLineWidth();
         float height = heightFactor * settings.getHeight();
         text.setLocalTranslation(width, height, 0);
     }
 
+    /**
+     * Getter for the players.
+     * Used in testing ONLY.
+     * @return the players hashmap.
+     */
+    public HashMap<PlatformPosition, BitmapText> getPlayers() {
+        return players;
+    }
```

Comment:
You can make this method protected.
While it still doesn't completely ensure that other classes in the same package (other than test classes) won't touch it, at least classes in other packages definately won't use it.